lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -104,7 +104,7 @@ def strip_coderay_prefix(hash)
 
         private
         def modernize_coderay_config
-          if highlighter == "coderay"
+          if !@config["coderay"].empty?
```

Comment:
I can't speak to the wider implications of this change, but as far as style goes, could we use `unless` instead of `if !`?
I think Rubocop should be complaining about this.